files/settings-default.cfg

```diff
@@ -280,6 +280,10 @@ force per pixel lighting = false
 # Setting this option to 'false' results in more realistic lighting.
 clamp lighting = true
 
+# Controls fog calculation. When disabled, the visible distance is larger on screen edges.
+# Enabling the setting makes fog distance accurate all around the player.
+accurate fog = false
```

Comment:
I really think this setting should be in the `[Fog]` section rather than the `[Shaders]` section. It requires shaders in order to work, but it's the fog that it affects.